<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>金融业务数据交换文件查看器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_open_fund_data">
       <attribute name="title">
        <string>开放式基金业务数据交换文件</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>文件名</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3" colspan="3">
           <widget class="QLineEdit" name="lineEdit_file_type"/>
          </item>
          <item row="5" column="6">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="2">
           <widget class="QComboBox" name="comboBox_interface_version"/>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QLineEdit" name="lineEdit_interface_file_path"/>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>发送人</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lineEdit_sender"/>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>接收人</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLineEdit" name="lineEdit_receiver"/>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="button_browse_file">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>接口文件</string>
            </property>
           </widget>
          </item>
          <item row="8" column="3">
           <widget class="QPushButton" name="button_search">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item row="6" column="3" colspan="3">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>文件类型码</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEdit_filename"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>文件日期</string>
            </property>
           </widget>
          </item>
          <item row="8" column="5">
           <widget class="QPushButton" name="button_export">
            <property name="text">
             <string>导出</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_search"/>
          </item>
          <item row="7" column="0">
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>TA代码</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3" colspan="3">
           <widget class="QLineEdit" name="lineEdit_ta_no"/>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="checkBox_remove_blank">
            <property name="text">
             <string>去掉空格</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>文件版本号</string>
            </property>
           </widget>
          </item>
          <item row="8" column="4">
           <widget class="QPushButton" name="button_restore">
            <property name="text">
             <string>恢复</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>#</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_monetary_fund_data">
       <attribute name="title">
        <string>货币基金T+0对账文件</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="5">
           <widget class="QPushButton" name="button_export_monetary">
            <property name="text">
             <string>导出</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QPushButton" name="button_browse_monetary_fund_t0_file">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="6">
           <widget class="QLineEdit" name="lineedit_monetary_fund_t0_file_path"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>对账文件</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_remove_blank_monetary">
            <property name="text">
             <string>去掉空白</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_search_monetary"/>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="button_search_monetary">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="button_restore_monetary">
            <property name="text">
             <string>恢复</string>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_monetary_fund_t0"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_ysstech_data">
       <attribute name="title">
        <string>赢时胜金手指文件</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>文件路径</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>文件类型</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>解析文件</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>保存文件</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionContent"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionContent">
   <property name="text">
    <string>内容</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_interface_file_path</tabstop>
  <tabstop>button_browse_file</tabstop>
  <tabstop>lineEdit_filename</tabstop>
  <tabstop>comboBox_interface_version</tabstop>
  <tabstop>lineEdit_ta_no</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>lineEdit_sender</tabstop>
  <tabstop>lineEdit_receiver</tabstop>
  <tabstop>lineEdit_file_type</tabstop>
  <tabstop>checkBox_remove_blank</tabstop>
  <tabstop>lineEdit_search</tabstop>
  <tabstop>button_search</tabstop>
  <tabstop>button_export</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>lineedit_monetary_fund_t0_file_path</tabstop>
  <tabstop>button_browse_monetary_fund_t0_file</tabstop>
  <tabstop>tableWidget_monetary_fund_t0</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
